qemu/tests/blockdev_commit_stress.py

```diff
@@ -0,0 +1,51 @@
+from virttest import utils_test
+
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitStress(BlockDevCommitTest):
+    def run_stress_test(self):
+        self.stress_test = utils_test.VMStress(self.main_vm, "stress",
+                                               self.params)
+        self.stress_test.load_stress_tool()
+
+    def stress_running_check(self):
+        if not self.stress_test.app_running():
+            self.test.error("Stress app does not running as expected")
+
+    def pre_test(self):
```

Comment:
what about:

def create_snapshots(...):
self.run_stress_test()
super(BlockdevCommitStress, self).create_snapshot(...)

def commit_snapshots():
super().commit_snapshots()
stress running check

Or we can modify base this way:
self.before_snapshot()
self.create_snapshots()
self.after_snapshot()

self.before_commit()
self.commit()
self.after_commit()
the before and after functions are empty functions which will be implemented in child.

But I think it's OK to rewrite the create_snapshots and commit_snapshots to do the 'before' and 'after' functions, just like what you did before.

Reply:
The second suggestion is perfect, will improve base later, but not this time. Thanks for you suggestion.